packages/sqlite_async/test/native/basic_test.dart

```diff
@@ -100,6 +105,122 @@ void main() {
       print("${DateTime.now()} done");
     });
 
+    test('with all connections', () async {
```

Comment:
Can we make this one a shared test that also runs on the web? It looks like we'd just have to remove the `expect(readers.length, 3);` check when we're running on the web because there are no separate readers.